README.md

```diff
@@ -24,3 +24,15 @@ While ComputerCraft will no longer be actively developed by Daniel Ratcliffe, yo
 * It does not create bugs!
 
 The pull requests most likely to be accepted are those which fix bugs, simplify code, or make the mod compatible with newer versions of Minecraft.
+
+## Building
+
+To build ComputerCraft, follow these steps:
+
+1. `git clone` this repository onto your machine.
+2. `cd` to it.
+3.  Run `./deploy.sh` on Unix, or `gradlew.bat build` on Windows.
```

Comment:
This isn't good advice, as gradlew.bat build does not do the same things as deploy.sh yet